src/app/shared/components/button/button.component.ts

```diff
@@ -1,16 +1,19 @@
-import { Component } from '@angular/core';
+import { Component, Input } from '@angular/core';
 import { FaIconLibrary } from '@fortawesome/angular-fontawesome';
-import { fas } from '@fortawesome/free-solid-svg-icons';
+import { fas, IconName, IconPrefix } from '@fortawesome/free-solid-svg-icons';
```

Comment:
I think loading the full icon package is not necessary. In fact, we are using only two of the icon pack as of now. Can you think if a solution to only include the .svg icons that are being used?

Reply:
done!